common/djangoapps/third_party_auth/models.py

```diff
@@ -379,6 +392,9 @@ class SAMLProviderConfig(ProviderConfig):
     attr_email = models.CharField(
         max_length=128, blank=True, verbose_name="Email Attribute",
         help_text="URN of SAML attribute containing the user's email address[es]. Leave blank for default.")
+    attr_country = models.CharField(
+        max_length=128, blank=True, verbose_name="Email Attribute",
+        help_text="URN of SAML attribute containing the user's ISO-3166-1 Alpha-2 country code. Leave blank for default.")
```

Comment:
@bradenmacdonald Ah this fact slipped by me. I think that since the other attributes do actually default to something, this should too. We can define it as a constant at the top there.